### PR TITLE
Preparing for the journey from ECS-to-EKS

### PR DESCRIPTION
**Pull request by drmdrew**
Here are the markdown slides for my _Preparing-ECS-to-EKS_ presentation.

*Note*: I created this branch for my presentation back when I thought I was giving it in July but that was cancelled due to the indoor rain delay!

### CODE REVIEW

File: 2018-07-25-Preparing-ECS-to-EKS/slides.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+<img src="images/mec-nalgene-bottle.png" style="opacity: 0.75; position: absolute; bottom: 50px; right: 0px"></img>
+ 
+---
+# Where are we starting from?
+
+* not an expert in ECS or EKS/k8s...
+* working with a project with microservices running in ECS
+* using a simplified environment for experimentation:
+  * [github.com/drmdrew/ecs-eks-demo][1]
```
Comment (daneroo) [inline 2018-07-25-Preparing-ECS-to-EKS/slides.md:25]: In the live slides, this link points to https://aws.amazon.com/blogs/opensource/vpc-cni-plugin-v1-1-available/
Probably something I did, just a heads up. The other links are fine.

**Comment by daneroo** [comment]
Looks great, can't wait to see the live version.
The EKS/Kubernetes Challenges pages is packed, could be it's own deck!
Cheers.

**Comment by daneroo** [comment]
Links fixed.

**Comment by jimareed** [comment]
Looks good!

**Pull request merged**